### ISSUE TITLE
(Core/Script): Sekeram gender target for locale

### ISSUE THREAD

**Issue by Maurowin** [azerothcore/azerothcore-wotlk#2864]
##### SMALL DESCRIPTION:

I don't know if it's a general problem to test but I had already spotted a problem with selecting text based on gender. You see the two choices displayed instead of selecting the right word for the gender.

![Sans titre](https://user-images.githubusercontent.com/31663279/78974230-d10a6000-7b11-11ea-9fde-c3887420c238.png)

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The word must be chosen according to gender and not show both possibilities.

##### STEPS TO REPRODUCE THE PROBLEM:

1. 
2. 
3. 

##### BRANCH(ES): Master

master

**Comment by Kitzunu** [comment]
It is actually quite easy to fix, it is due to wrong targeting in SAI

**Comment by Kitzunu** [comment]
What is the creature name or entry? I dont know french

**Comment by Maurowin** [comment]
Its The Prophet Skeram in AQ40 : https://wowwiki.fandom.com/wiki/The_Prophet_Skeram

**Comment by Kitzunu** [comment]
Oh it is a boss, boss_skeram, then something is wrong in targeting in the script. and I don't have enough knowledge yet of C++ to fix that

**Comment by Kitzunu** [comment]
Hmm... I cannot find this line in the DB 
```sql
SELECT * FROM creature_text WHERE creatureid = 15263
```
Also in https://wowwiki.fandom.com/wiki/The_Prophet_Skeram I cannot see any gender specific lines.

Is this some french local only text?

**Comment by Maurowin** [comment]
Yes i guess, did u see on local field?

**Comment by Kitzunu** [comment]
yea, creature_text_locale dont have any entry

**Comment by Maurowin** [comment]
I think its in Broadcast text local

**Comment by Kitzunu** [comment]
I will check

**Comment by Kitzunu** [comment]
Yes
```sql
SELECT * FROM broadcast_text_locale WHERE id IN (11445) AND locale = 'frFR';
```

**Comment by Maurowin** [comment]
Yes this one !

**Comment by Kitzunu** [comment]
So we need to check here who triggers it
```cpp
            void EnterCombat(Unit* /*who*/)
            {
                _EnterCombat();
                events.Reset();

                events.ScheduleEvent(EVENT_ARCANE_EXPLOSION, urand(6000, 12000));
                events.ScheduleEvent(EVENT_FULLFILMENT, 15000);
                events.ScheduleEvent(EVENT_BLINK, urand(30000, 45000));
                events.ScheduleEvent(EVENT_EARTH_SHOCK, 2000);

                Talk(SAY_AGGRO);
            }
```
I don't know French at all, but I don't feel like it makes any sense to check for the gender of who aggroes the boss.

In Enligh the line is "Are you so eager to die? I will be happy to accommodate you..." where "you" I think means the whole group.

What does pressé:pressée mean? It looks the same to me

**Comment by Maurowin** [comment]
yes to whole group. Its mean : pressed
@BarbzYHOOL can explain better than me, he speak better english than me either :D

**Comment by Kitzunu** [comment]
I mean, is it neccessary with gender specific?

**Comment by BarbzYHOOL** [comment]
yes it's specific to french, spanish too

adjectives depends on the gender unlike in english "pressé = male" "pressée = female"

**Comment by Kitzunu** [comment]
is there no generic one? Because what if the group in the raid has both men and women?

**Comment by BarbzYHOOL** [comment]
then it would have to speak in plural form "êtes-vous pressés..." and not target anybody

**Comment by Kitzunu** [comment]
that makes more sense imo... taking into account that it could be both... But I dont know how it was in retail :(

**Comment by Maurowin** [comment]
@BarbzYHOOL @Kitzunu No, it stays in the singular, it's the first one who pulls that's concerned.

**Comment by Kitzunu** [comment]
Unfortunatly this is out of my league...
I don't know what to look for since I Don't know another script that checks something like this :(

**Comment by Maurowin** [comment]
Iam not Dev, but maybe it can help for this issue or not.
@BarbzYHOOL @Viste 
https://github.com/TrinityCore/TrinityCore/pull/23520/files
